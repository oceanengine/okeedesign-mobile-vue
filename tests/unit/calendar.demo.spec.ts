import Demo from '@src/components/calendar/demo/index.vue';
import { snapshotDemo } from '../demo';

snapshotDemo(Demo);
