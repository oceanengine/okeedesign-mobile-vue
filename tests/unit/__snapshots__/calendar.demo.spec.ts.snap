// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="demo calendar">
  <div class="o-header--wrapper">
    <div class="o-header o-header--default o-header--fixed o-hairline--bottom">
      <div class="o-header__left"><i class="o-header__icon" style="width: 1.2rem; height: 1.2rem;">test-file-stub</i></div>
      <div class="o-header__title">
        <div></div>
      </div>
      <div class="o-header__right"></div>
    </div>
  </div>
  <div>
    <div class="bui-doc-demo-block__card">
      <div class="o-cell o-cell--clickable">
        <div class="o-cell__title"><span></span></div><i class="o-cell__right-icon">test-file-stub</i>
      </div>
      <div class="o-cell o-cell--clickable">
        <div class="o-cell__title"><span></span></div><i class="o-cell__right-icon">test-file-stub</i>
      </div>
      <div class="o-cell o-cell--clickable">
        <div class="o-cell__title"><span></span></div>
        <div class="o-cell__value o-cell__value--alone"><span class="custom-value">
          2/2 - 2/28
        </span></div><i class="o-cell__right-icon">test-file-stub</i>
      </div>
    </div>
    <!---->
    <!---->
    <!---->
  </div>
  <div>
    <div class="o-cell o-cell--clickable">
      <div class="o-cell__title"><span></span></div><i class="o-cell__right-icon">test-file-stub</i>
    </div>
    <!---->
    <div class="o-cell o-cell--clickable">
      <div class="o-cell__title"><span></span></div>
      <div class="o-cell__value o-cell__value--alone"><span class="custom-value">

      </span></div><i class="o-cell__right-icon">test-file-stub</i>
    </div>
    <!---->
  </div>
  <div></div>
  <div class="o-calendar">
    <header class="o-calendar__header">
      <div class="o-calendar__title-bar">
        <div class="o-calendar__title">选择时间</div>
        <div class="o-calendar__header-left"><button class="o-calendar__button">取消</button></div>
        <div class="o-calendar__header-right"><button class="o-calendar__button">确定</button></div>
      </div>
      <div class="o-calendar__week">
        <div class="o-calendar__day o-calendar__day--rest">日</div>
        <div class="o-calendar__day">一</div>
        <div class="o-calendar__day">二</div>
        <div class="o-calendar__day">三</div>
        <div class="o-calendar__day">四</div>
        <div class="o-calendar__day">五</div>
        <div class="o-calendar__day o-calendar__day--rest">六</div>
      </div>
    </header>
    <div class="o-calendar__body">
      <div class="o-calendar__month-wrap">
        <div class="o-calendar__month o-calendar__month--0">
          <div class="o-calendar__month-bar">2020年2月</div>
          <div class="o-calendar__date-wrap">
            <div class="o-calendar__date-placeholder o-calendar__date-placeholder--6"></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">1</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest o-calendar__date--selected">2</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">3</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">4</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">5</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">6</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">7</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">8</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">9</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">10</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">11</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">12</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">13</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">14</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">15</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">16</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">17</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">18</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">19</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">20</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">21</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">22</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">23</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">24</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">25</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">26</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">27</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">28</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">29</button></div>
          </div>
        </div>
        <div class="o-calendar__month o-calendar__month--1">
          <div class="o-calendar__month-bar">2020年3月</div>
          <div class="o-calendar__date-wrap">
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">1</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">2</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">3</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">4</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">5</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">6</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">7</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">8</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">9</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">10</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">11</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">12</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">13</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">14</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">15</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">16</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">17</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">18</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">19</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">20</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">21</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">22</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">23</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">24</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">25</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">26</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">27</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">28</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date o-calendar__date--rest">29</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">30</button></div>
            <div class="o-calendar__cell"><button class="o-calendar__date">31</button></div>
          </div>
        </div>
      </div>
    </div>
    <div class="o-calendar__bottom-float">
      <!---->
    </div>
  </div>
</div>
`;
